lib/core/test/unit/Cardano/Wallet/Primitive/CoinSelection/RandomSpec.hs

```diff
@@ -225,6 +228,43 @@ spec = do
                 , txOutputs = 2 :| []
                 })
 
+    describe "Coin selection : randomWithoutTxOut algorithm unit tests" $ do
+        coinSelectionWithoutTxOutUnitTest randomWithoutTxOut ""
+            (Right $ CoinSelectionResult
+                { rsInputs = [1]
+                , rsChange = [1]
```

Comment:
I wonder if I think about it properly, in coin selection I end up here with `input = [1], change=[1], output=[]`, right?